front/components/app/DatasetView.tsx

```diff
@@ -144,9 +144,20 @@ export default function DatasetView({
     }
 
     return valid;
-  };
+  }, [dataset?.name, datasetName, datasets]);
```

Comment:
I'm not sure this is correct, because `datasets` is an array, so its reference could be the same even if the array was modified, no ? Is it required by `exhaustive-deps` that this must be a `useCallback` ?